src/garage/torch/policies/stochastic_policy.py

```diff
@@ -25,12 +26,13 @@ def get_action(self, observation):
                     * np.ndarray[float]: Mean of the distribution
                     * np.ndarray[float]: Standard deviation of logarithmic
                         values of the distribution.
-
         """
         with torch.no_grad():
             if not isinstance(observation, torch.Tensor):
                 observation = torch.as_tensor(observation).float().to(
                     global_device())
+            if isinstance(self._env_spec.observation_space, akro.Image):
```

Comment:
This is kinda a really weird place to put this logic. I would expect it to be in `CNNModule`, since it's already essentially image-specific. It's not a big issue either way though.